srcpkgs/yabridgectl/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'yabridgectl'
+pkgname=yabridgectl
+version=4.0.2
+revision=1
+archs="x86_64"
```

Comment:
why?

Reply:
this was not resolved, unmarking as resolved

Reply:
Why do you keep marking this as resolved? I would like to see a reason first. Does it not work on musl? on i686? Are cross builds broken? Why?